core/templates/dev/head/pages/header_js_libs.html

```diff
@@ -3,6 +3,25 @@
 <% if (webpackConfig.mode == 'production') { %>
   <script src="/assets/hashes.js"></script>
 <% } %>
+<script>
+  // This is kept here since the values here are required before
+  // angular is bootstrapped.
+  (function () {
+    var pages = [
+      'create', 'creator_dashboard', 'explore', 'review', 'review_test',
+      'skill_editor', 'story_editor', 'topic_editor'
+    ];
+    var hasAdditionalModule = false;
+    pages.forEach(function (page) {
```

Comment:
`pageNames.forEach(function(pageName)) {`

(in general, distinguish between the name of a thing, and the thing)